app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -52,18 +58,27 @@ class MS.Views.LoanChartsView extends Backbone.View
     chart.draw(data, options)
 
   breakevenProductProfitChart: ->
-    data = new google.visualization.DataTable();
+    data = new google.visualization.DataTable()
     data.addColumn 'string', I18n.t('loan.breakeven.product')
     data.addColumn 'number', I18n.t('loan.breakeven.profit')
+    options = @defaultChartOptions
 
+    negativeNums = false
```

Comment:
The bar chart breaks the legend of the other two charts and has been removed. If we need to address negative values (which would only occur if it costs more to produce an item than it what the item brings in with its revenue), a separate issue to address this can be filed.